server.php

```diff
@@ -8,11 +9,14 @@
 $uri = urldecode(
     parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH)
 );
-// This file allows us to emulate Apache's "mod_rewrite" functionality from the
-// built-in PHP web server. This provides a convenient way to test a Laravel
-// application without having installed a "real" web server software here.
-if ($uri !== '/' and file_exists(__DIR__.'/'.$uri))
-{
+
+/**
+ * This file allows us to emulate Apache's "mod_rewrite" functionality from the
+ * built-in PHP web server. This provides a convenient way to test a Laravel
+ * application without having installed a "real" web server software here.
+ */
+if ($uri !== '/' && file_exists(__DIR__.'/'.$uri)) {
```

Comment:
@ayumi-cloud Would you mind adding some spacing around the concatenation parts?
```suggestion
if ($uri !== '/' && file_exists(__DIR__ . '/' . $uri)) {
```